LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2020 PERSISTENCE TECHNOLOGIES PTE. LTD.
```

Comment:
This must be put in the copyrighted files and not the licence itself, afaik.

Reply:
reference,
https://github.com/cosmos/cosmos-sdk/blob/master/LICENSE#L192
or 
https://github.com/ovrclk/akash/blob/master/LICENSE#L178

can close PR, and the issue both, been pending since a long time.

Reply:
``` 
      To apply the Apache License to your work, attach the following
      boilerplate notice, with the fields enclosed by brackets "[]"
      replaced with your own identifying information. (Don't include
      the brackets!)  The text should be enclosed in the appropriate
      comment syntax for the file format. We also recommend that a
      file or class name and description of purpose be included on the
      same "printed page" as the copyright notice for easier
      identification within third-party archives.
      ```